chaoslib/openebs/cstor_pool_container_kill.yml

```diff
@@ -0,0 +1,69 @@
+- name: Derive PV from application PVC 
+  shell: >
+    kubectl get pvc {{ app_pvc }}
+    -o custom-columns=:spec.volumeName -n {{ app_ns }}
+    --no-headers
+  args:
+    executable: /bin/bash
+  register: pv
+
+- name: Randomly select the pool deployment from cvr
+  shell: >
+    kubectl get cvr -n {{ operator_ns }}
+    -l openebs.io/persistent-volume={{ pv.stdout }} --no-headers
+    -o=jsonpath='{range .items[*]}{.metadata.labels.cstorpool\.openebs\.io\/name}{"\n"}{end}' |
+    shuf -n1 | awk '{print $1}'
+  args:
+    executable: /bin/bash
+  register: pool_deployment
+
+- name: Get the pod of pool deployment
+  shell: >
+    kubectl get pods -n {{ operator_ns }} |
+    grep {{ pool_deployment.stdout }} | grep -w "Running" | awk '{print $1}'
```

Comment:
We should grep here only for pool deployment to get the pod name because of following:
1. Anyways we are checking for pool pod running status in next task.
2. If for any any reason the pod corresponding to the deployment is not in running state this shell will give non-zero return.

Reply:
@shashank855  done